images/linux/scripts/installers/bicep.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+################################################################################
+##  File:  bicep.sh
+##  Desc:  Installs bicep cli
+################################################################################
+
+source $HELPER_SCRIPTS/install.sh
+
+# Install Bicep CLI
+download_with_retries "https://github.com/Azure/bicep/releases/latest/download/bicep-linux-x64" "." "bicep.bin"
```

Comment:
nitpick: use dst param and remove mv statement
```
download_with_retries "https://github.com/Azure/bicep/releases/latest/download/bicep-linux-x64" "/usr/local/bin" "bicep"
chmod +x /usr/local/bin/bicep
```

Reply:
I'd prefer to stay in line with our official install steps here: https://github.com/Azure/bicep/blob/main/docs/installing.md#linux